Library/Homebrew/utils/analytics.sh

```diff
@@ -0,0 +1,67 @@
+setup-analytics() {
+  if [[ -z "$HOMEBREW_ANALYTICS" || -n "$HOMEBREW_NO_ANALYTICS" ]]
+  then
+    return
+  fi
+
+  # User UUID file. Used for Homebrew user counting. Can be deleted and
+  # recreated with no adverse effect (beyond our user counts being inflated).
+  HOMEBREW_ANALYTICS_USER_UUID_FILE="$HOME/.homebrew_analytics_user_uuid"
+  if [[ -r "$HOMEBREW_ANALYTICS_USER_UUID_FILE" ]]
+  then
+    HOMEBREW_ANALYTICS_USER_UUID="$(<"$HOMEBREW_ANALYTICS_USER_UUID_FILE")"
+  else
+    HOMEBREW_ANALYTICS_USER_UUID="$(uuidgen)"
+    echo "$HOMEBREW_ANALYTICS_USER_UUID" > "$HOMEBREW_ANALYTICS_USER_UUID_FILE"
+  fi
+  export HOMEBREW_ANALYTICS_ID="UA-75654628-1"
+  export HOMEBREW_ANALYTICS_USER_UUID
+}
+
+report-analytics-screenview-command() {
+  if [[ -z "$HOMEBREW_ANALYTICS" || -n "$HOMEBREW_NO_ANALYTICS" ]]
+  then
+    return
+  fi
+
+  # Don't report non-official commands.
+  if ! [[ "$HOMEBREW_COMMAND" = "bundle"   ||
+          "$HOMEBREW_COMMAND" = "cask"     ||
+          "$HOMEBREW_COMMAND" = "services" ||
+          -f "$HOMEBREW_LIBRARY/Homebrew/cmd/$HOMEBREW_COMMAND.rb"     ||
+          -f "$HOMEBREW_LIBRARY/Homebrew/cmd/$HOMEBREW_COMMAND.sh"     ||
+          -f "$HOMEBREW_LIBRARY/Homebrew/dev-cmd/$HOMEBREW_COMMAND.rb" ||
+          -f "$HOMEBREW_LIBRARY/Homebrew/dev-cmd/$HOMEBREW_COMMAND.sh" ]]
+  then
+    return
+  fi
+
+  local ARGS=(
+    --max-time 3 \
+    --user-agent "$HOMEBREW_USER_AGENT_CURL" \
+    -d v="1" \
+    -d tid="$HOMEBREW_ANALYTICS_ID" \
+    -d cid="$HOMEBREW_ANALYTICS_USER_UUID" \
+    -d aip="1" \
+    -d an="$HOMEBREW_PRODUCT" \
+    -d av="$HOMEBREW_VERSION" \
+    -d t="screenview" \
+    -d cd="$HOMEBREW_COMMAND" \
+  )
+
+  if [ -z "$HOMEBREW_ANALYTICS_DEBUG" ]
+  then
+    # Send the to-be-executed command as an "App Screen View". Anonymise the IP
+    # address (aip=1) and don't send or store any personally identifiable
+    # information.
+    # https://github.com/Homebrew/brew/blob/master/share/doc/homebrew/Analytics.md
+    # https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide#screenView
+    # https://developers.google.com/analytics/devguides/collection/protocol/v1/parameters
```

Comment:
This seems similarly misplaced like the corresponding comment in the Ruby code. (Doesn't it also apply to the debug/`else` branch of this `if`?)